.github/workflows/homework_04_19_teodor_dishanski.yaml

```diff
@@ -0,0 +1,53 @@
+name: Homework 04 - 19 Teodor Dishanski 
+
+on:
+  push:
+    branches: [ main ]
+    paths: 
+      - homework_04/19_teodor_dishanski/**
+  pull_request:
+    branches: [ main ]
+    paths: 
+      - homework_04/19_teodor_dishanski/**
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Dependencies from Requirements.txt
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Install Additional Linters
+      run: |
+        pip install pylint
+        pip install pylama
+        pip install pycodestyle
+        pip install pydocstyle
+        pip install mypy
+    - name: Run All Unit Tests
+      run: |
+        cd homework_04/19_teodor_dishanski
+        pytest
+    - name: Run All Linters
+      run: |
+        cd homework_04/19_teodor_dishanski
+        pylint *.py
+        pylama *.py
+        pycodestyle *.py
+        pydocstyle *.py
+        mypy *.py
```

Comment:
Too many linters. Maybe this is little bit unnecessary

Reply:
Nice work